app/views/user_mailer/team_member_reminder.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<h4>Dear <%= @recipient.fullname %></h4>
+
+<p>Please view your current initiative status below:</p>
+
+    <% @recipient.initiatives.incompleted.each do |initiative| %>
+      <p><%= link_to initiative.title, root_url %></p>
```

Comment:
Does it seem helpful to have the Target Date here so they can quickly decide if they need to review it right now?

Reply:
I have added the target date